src/model/model.js

```diff
@@ -382,6 +388,48 @@ export default class Model {
 		this.stopListening();
 	}
 
+	/**
+	 * Creates deltas from given {@link module:engine/model/documentfragment~DocumentFragment}.
+	 *
+	 * Returned array contains exactly one {@link module:engine/model/operation/insertoperation~InsertOperation} which
+	 * inserts all the nodes and zero or more {@link module:engine/model/operation/markeroperation~MarkerOperation}, depending
+	 * on {@link module:engine/model/documentfragment~DocumentFragment#markers markers set on the document fragment}.
+	 *
+	 * @param {module:engine/model/documentfragment~DocumentFragment} documentFragment Document fragment to be converted to deltas.
+	 * @param {String} [rootName] Name of a root element which will be base for the operations. The returned insert
+	 * operation will insert the nodes inside the given root. If not set, the main root will be used.
+	 * @returns {Array.<module:engine/model/operation/operation~Operation>} Operations which, when applied, will recreate given
+	 * `documentFragment` inside the specified root.
+	 */
+	convertToDeltas( documentFragment, rootName ) {
```

Comment:
If you can think of some other name, which avoids the word "convert", I'm sure @pjasiun would be happy :P

Reply:
Yep. And `transformToDeltas` won't be good either.

Reply:
In the F2F talk, we agreed it will have no parameters and will serialize model to deltas. It can be called `getAsDeltas`